charts/blaze/templates/deployment-frontend.yaml

```
@@ -0,0 +1,68 @@
+{{- if .Values.frontend.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "blaze.fullname" . }}-frontend
+  labels:
+    {{- include "blaze.labels" . | nindent 4 }}
+    app.kubernetes.io/component: frontend
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" (dict "value" .Values.commonAnnotations "context" $) | nindent 4 }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "blaze.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: frontend
+  replicas: {{ .Values.frontend.replicaCount }}
+  template:
+    metadata:
+      {{- with .Values.frontend.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "blaze.selectorLabels" . | nindent 8 }}
+        app.kubernetes.io/component: frontend
+    spec:
+      {{- with .Values.frontend.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.podSecurityContext }}
```

```suggestion
      {{- with .Values.frontend.podSecurityContext }}
```

Also frontend dedizierte securityContext könnte man überlegen, wäre aber wahrscheinlich unnötig redundant momentan und kann man immer noch später ergänzen.

---

Hatte es mit Absicht weggelassen aber ich seh jetzt auch keinen Grund dafür das nicht zu trennen und mit passenden default Werten zu hinterlegen. Übernehm ich.